### PR TITLE
User statistic chart fix

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/user/userActions.js [hunk 1]
```diff
@@ -258,3 +258,11 @@ export const inactivateCampaign = (company, inactivatePermlink) => (
       .catch(error => reject(error));
   });
 };
+
+export const GET_FORECAST_CHART_CONDITION = '@users/GET_FORECAST_CHART_CONDITION';
+
+export const setForecastAccuracyChartCondition = () => dispatch =>
```
Comment (milklim) [inline src/client/user/userActions.js]: get or set?

File: src/client/reducers.js [hunk 2]
```diff
@@ -134,6 +134,7 @@ export const getIsLoadingNotifications = state => fromUser.getIsLoadingNotificat
 export const getFetchFollowListError = state => fromUser.getFetchFollowListError(state.user);
 export const getLatestNotification = state => fromUser.getLatestNotification(state.user);
 export const getUserLocation = state => fromUser.getUserLocation(state.user);
+export const  getUserForecastAccuracyChartCondition = state => fromUser.getUserForecastAccuracyChartCondition(state.user);
```
Comment (milklim) [inline src/client/reducers.js]: rename selectors

File: src/client/user/userReducer.js [hunk 3]
```diff
@@ -22,6 +23,10 @@ const initialState = {
   latestNotification: {},
   loadingNotifications: false,
   fetchFollowListError: false,
+  statistics:
+    {
+      isForecastAccuracyChart: false
```
Comment (milklim) [inline src/client/user/userReducer.js]: unclear property name

File: src/client/user/userActions.js [hunk 4]
```diff
@@ -258,3 +258,11 @@ export const inactivateCampaign = (company, inactivatePermlink) => (
       .catch(error => reject(error));
   });
 };
+
+export const SET_ACCURACY_CHART_CONDITION = '@users/SET_ACCURACY_CHART_CONDITION';
```
Comment (milklim) [inline src/client/user/userActions.js]: @users => @user
ACCURACY_CHART_CONDITION => ACCURACY_CHART_LOADED

File: src/client/user/userActions.js [hunk 5]
```diff
@@ -258,3 +258,11 @@ export const inactivateCampaign = (company, inactivatePermlink) => (
...
+
+export const setAccuracyChartCondition = () => dispatch =>
```
Comment (milklim) [inline src/client/user/userActions.js]: rename to setAccuracyChartLoaded

File: src/client/user/userReducer.js [hunk 6]
```diff
@@ -22,6 +22,9 @@ const initialState = {
   latestNotification: {},
   loadingNotifications: false,
   fetchFollowListError: false,
+  statistics: {
+    isAccuracyChart: false,
```
Comment (milklim) [inline src/client/user/userReducer.js]: to 'isAccuracyChartLoaded'

File: src/client/reducers.js [hunk 7]
```diff
@@ -134,6 +134,7 @@ export const getIsLoadingNotifications = state => fromUser.getIsLoadingNotificat
 export const getFetchFollowListError = state => fromUser.getFetchFollowListError(state.user);
 export const getLatestNotification = state => fromUser.getLatestNotification(state.user);
 export const getUserLocation = state => fromUser.getUserLocation(state.user);
+export const getAccuracyChartCondition = state => fromUser.getAccuracyChartCondition(state.user);
```
Comment (milklim) [inline src/client/reducers.js]: rename 'getAccuracyChartCondition' to 'getAccuracyChartLoaded'

Comment (konstantin1390) [reply src/client/reducers.js]: +

File: src/client/user/userActions.js (hunk 4 above)
Comment (konstantin1390) [reply src/client/user/userActions.js]: +

File: src/client/user/userActions.js (hunk 5 above)
Comment (konstantin1390) [reply src/client/user/userActions.js]: +

File: src/client/user/userReducer.js (hunk 6 above)
Comment (konstantin1390) [reply src/client/user/userReducer.js]: +

**Pull request merged**